pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -196,6 +205,35 @@ func (i *Instance) IsTopLevelInstance() bool {
 	return !i.IsChildInstance()
 }
 
+const (
+	planInProgressReason   = "PlanInProgress"
+	planInFatalErrorReason = "PlanInFatalError"
+	resourceNotReadyReason = "ResourceNotReady"
+	resourcesReadyReason   = "ResourcesReady"
+	readyConditionType     = "Ready"
+)
+
+func (i *Instance) SetReadinessTrue(msg string) {
+	condition := metav1.Condition{Type: readyConditionType, Status: metav1.ConditionTrue, Message: msg, Reason: resourcesReadyReason}
+	meta.SetStatusCondition(&i.Status.Conditions, condition)
+}
+
+func (i *Instance) SetNotReady(msg string) {
```

Comment:
nit:
```suggestion
func (i *Instance) SetReadinessFalse(msg string) {
```

Reply:
well okay, I liked mine more :D

Reply:
I'm ok with both naming schemes when applied consistently consistently. Either:
```
func SetReady
func SetNotReady
```
or
```
func SetReadinessFalse
func SetReadinessTrue
```
combining both is just confusing 🤷